russian/call-mom/call-mom.md

```diff
@@ -1 +1,7 @@
 Вы позвонили своей маме и рассказали о том, что происходит.
+Она вам сообщила, что Вас на самом деле похитила группа преступников и погрузила в сон одного из них - так называемого Архитектора. Как только она начала говорить зачем - звонок оборвался
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.